### ISSUE TITLE
StackOverflow Profile Top Tags section displaying white background

### ISSUE THREAD

**Issue by spybug** [StylishThemes/StackOverflow-Dark#129]
When viewing the profile tab section on stackoverflow, it appears that the top tags section has each tag cell with a white background which makes it hard to read and different then before.

* **Browser**: Chrome v69
* **Operating System**: Windows 10
* **Screenshot**: ![screenshot](https://i.imgur.com/nkVDG6S.png)

* **HTML of the section where the issue occurs**:

````html
<div class="grid--cell">
                    <div class="grid gs8">
                            <div class="grid grid__fl1 p8 bg-black-025 bar-sm grid__allcells12" title=" Gave 34 non-wiki answers with a total score of 12.">
                                <div class="grid--cell">
                                    <a href="/search?q=user:7087143+[python]" class="post-tag" title="" rel="tag">python</a>

                                </div>
                                <div class="grid jc-end ml-auto">
                                    <div class="grid gs16 gsx">
                                        <div class="grid ai-center h100 fs-fine lh-xs fc-black-350">
                                            <span class="mr4 fw-bold tt-uppercase">Score</span>
                                            <span class="fc-medium fs-title">12</span>
                                        </div>
                                        <div class="grid ai-center h100 fs-fine lh-xs fc-black-350">
                                            <span class="mr4 fw-bold tt-uppercase">Posts</span>
                                            <span class="fc-medium fs-title">34</span>
                                        </div>
                                            <div class="grid ai-center h100 fs-fine lh-xs fc-black-350">
                                                <span class="mr4 fw-bold tt-uppercase">Posts %</span>
                                                <span class="fc-medium fs-title">72</span>
                                            </div>
                                    </div>
                                </div>
                            </div>
                    </div>
                </div>
````

**Comment by Mottie** [comment]
Hi @spybug!

I'm not seeing that same issue on my end... I think you might be using the version of the style provided by Userstyles.org - we've stopped updating that site.

If you would like to get the most recent update, please install a Stylus or xStyle, then install this style as a UserCSS - [see the readme](https://github.com/StylishThemes/StackOverflow-Dark#installation) for more details.

**Comment by spybug** [comment]
I made sure to delete and re-install the style from the github link. I have v3.0.7 installed: ![image](https://i.imgur.com/JMeOhV8.png)

However it still appears the same way on that profile page even after doing Shift + F5 to refresh the page.

**Comment by Mottie** [comment]
Derp, I was looking at my own profile and not a different users...

Anyway, it's been fixed in v3.0.8.
